engine/core/IgeTileMap2d.ts

```diff
@@ -31,7 +31,7 @@ export class IgeTileMap2d extends IgeEntity {
 	IgeTileMap2d = true;
 	_drawGrid?: boolean;
 	_highlightOccupied: boolean = false;
-	_highlightTileRect: boolean = false;
+	_highlightTileRect?: {x: number, y: number, width: number, height: number} = undefined;
```

Comment:
Oops, well spotted LOL. I wonder if this would be better defined as an `IgeRect`?